packages/inventory-insights/src/Insights.js

```diff
@@ -95,10 +95,14 @@ class InventoryRuleList extends Component {
 
     fetchKbaDetails = async (reportsData) => {
         const { filters, searchValue, rows } = this.state;
-        const kbaIds = reportsData.map(report => report.rule.node_id).filter(x => x).join(` OR `);
+        let nodeIdCount = 0;
+        const kbaIds = reportsData.map(report => {
+            report.rule.node_id && nodeIdCount++;
```

Comment:
This will include reports that will be filtered later on is that correct. I think that this can be slightly changes
```JS
const kbaIds = reportsData.map(({ rule }) => rule.node_id).filter(x => x);
const kbaDetailsFetch = (await API.get(
  `https://access.redhat.com/hydra/rest/search/kcs?q=id:(${
    kbaIds.join(` OR `)
  })&fl=view_uri,id,publishedTitle&rows=${
    kbaIds.length
  }`,
  {},
  { credentials: 'include' }
 )).data.response.docs;
```

Reply:
All we do with the filter there is kill blank entries for a clean join, but dig this, like the idea of no let, thanks for the 👁️ 🤗

Reply:
works out okie dokie with no ids and 1 id so force push incoming!